spec/features/switching_policies_spec.rb

```diff
@@ -1,17 +1,29 @@
 require "rails_helper"
 
 RSpec.feature "A user can switch policies" do
-  it "a user can switch to maths and physics after starting a student loan claim" do
-    start_student_loans_claim
-    visit new_claim_path(MathsAndPhysics.routing_name)
+  context "after starting a student loan claim" do
+    before do
+      start_student_loans_claim
+      visit new_claim_path(MathsAndPhysics.routing_name)
+    end
 
-    expect(page).to have_text(I18n.t("maths_and_physics.questions.teaching_maths_or_physics"))
+    scenario "a user can switch to maths and physics after starting a student loan claim" do
+      click_on "Start claim"
+
+      expect(page).to have_text(I18n.t("maths_and_physics.questions.teaching_maths_or_physics"))
+    end
   end
 
-  it "a user can switch to student loans after starting a maths and physics claim" do
-    start_maths_and_physics_claim
-    visit new_claim_path(StudentLoans.routing_name)
+  context "after starting a maths and physics claim" do
```

Comment:
I wonder whether we need to test with two different initial policies. I think we could probably just test switching from, say, Student Loans to Maths and Physics.